### ISSUE TITLE
(feat): Azure Managed Disk support

### ISSUE THREAD

**Issue by lukasmrtvy** [crossplane/crossplane#1665]
Would be nice to have support also for Azure Managed Disk as CRD.

Is possible to reference pre existing managed disk in PV:
https://docs.microsoft.com/en-us/azure/aks/azure-disk-volume#mount-disk-as-volume
https://github.com/Azure/aks-engine/issues/3648#issuecomment-665857839

Thanks

**Comment by negz** [comment]
Agreed! Thanks for the feature request; this helps us prioritise resources. I'm going to migrate this issue to the Azure provider repo.
